docs/getting-started/install-guide.md

```diff
@@ -111,24 +132,8 @@ Note: Windows 7/8/10 natively provides support for `.zip` files, therefore it is
 
 If you decide to download the `.tar.gz` file, it will require two separate extractions in some third-party application (7-zip, winRAR, etc..) to get to the actual binaries.
 
-## **Windows Installer** 
-
-The Windows Installer (`.msi` file) is located here: [https://github.com/decred/decred-binaries/releases](https://github.com/decred/decred-binaries/releases). It will install Paymetheus to your computer's Program Files folder. Installation is pretty straightforward, but instructions are provided below:
-
-1. Download the correct file:
-
-    For 32-bit computers, download the `decred_0.8.2-beta_x86.msi` file. <br />
-    For 64-bit computers, download the `decred_0.8.2-beta_x64.msi` file.
-
-2. Navigate to download location and open the `.msi` file.
-
-3. Follow the installation steps. Within this process you'll be prompted to accept an End-User License Agreement.
-
-4. After setup, the features should be installed to your `..\Program Files\Decred\` folder and accessible through the Start Menu (look for `Decred` in the Program list)
-
 -----------------------
 
-Paymetheus users can continue to [Paymetheus setup](/getting-started/user-guides/paymetheus.md)
+Paymetheus users can continue The Golden Path by visiting the [Paymetheus Setup](/getting-started/user-guides/paymetheus.md) guide.
```

Comment:
How will those on The Golden Path install their software if the install guide isn't included?